libs/services/src/events/hatom.events.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from "@nestjs/common";
+import { EventLog } from "apps/api/src/endpoints/events/entities";
+import { Address } from "@multiversx/sdk-core";
+import BigNumber from "bignumber.js";
+import { CsvRecordsService } from "../records";
+import moment from "moment";
+import { DataService } from "../data";
+import { TableSchema } from "apps/dune-simulator/src/endpoints/dune-simulator/entities";
+
+interface BorrowEvent {
+    eventName: string;
+    borrowerAddress: string;
+    amount: BigNumber;
+    newAccountBorrow: BigNumber;
+    newTotalBorrows: BigNumber;
+    newBorrowerIndex: BigNumber;
+}
+
+@Injectable()
+export class HatomEventsService {
```

Comment:
we only used one type of hatom events (the `borrow`). will we have different files/services for other events, such as lending/liquidation?

Reply:
yes, it will be cleaner this way. I will rename this service to -> HatomBorrowEventsService